### PR TITLE
Changed random name generation to use Get-Random rather than System.Random

### PR DESCRIPTION
**Pull request by Staples1887**
…andom

### Description of the Change
Changed random name generation to use Get-Random rather than System.Random

### Related Issue
#527

### CODE REVIEW

File: src/Private/Helpers.ps1 [hunk 1]
```diff
@@ -341,9 +341,8 @@ function Get-PodeWebRandomName
         $Length = 5
     )
 
-    $r =  [System.Random]::new()
     return [string]::Concat(@(foreach ($i in 1..$Length) {
-        [char]$r.Next(65, 90)
+        [char](Get-Random -Minimum 65 -Maximum 90)
     }))
```
Comment (Badgerati) [inline src/Private/Helpers.ps1]: You must have a much faster system than me, I always get different random names! 😂 

My only concern with `Get-Random` is that is can be 2-3x slower than the previous method of getting a random name. Out of interest, do you still get the same issue if you replace the whole block with the following?

```powershell
return [System.Security.Cryptography.RandomNumberGenerator]::GetHexString($Length)
```

If that works, then we might possible use your new method above for people running on PS5.1, and then the Crpyto RNG above for people on PS7.x as it'll be much faster 🤔

Comment (Staples1887) [reply src/Private/Helpers.ps1]: When I run it in pwsh 7.4 both your suggested change and the original seem to work. 😬  So it's just an issue when running 5.1.

Comment (Badgerati) [reply src/Private/Helpers.ps1]: Oooh, really? In which case we could just keep the original and do a check for PS5 then 😄 something like this, to use `Get-Random` on PS5, and the original on 6+:

```powershell
if ($PSVersionTable.PSVersion.Major -eq 5) {
   return [string]::Concat(@(foreach ($i in 1..$Length) {
      [char](Get-Random -Minimum 65 -Maximum 90)
   }))
}
else {
   $r = [System.Random]::new()
   return [string]::Concat(@(foreach ($i in 1..$Length) {
      [char]$r.Next(65, 90)
   }))
}
```

Comment (Staples1887) [reply src/Private/Helpers.ps1]: Changes made

**Comment by Badgerati** [comment]
LGTM! :)

**Pull request merged**